sdk/javascript/vectors/catbuffer.js

```diff
@@ -0,0 +1,98 @@
+const nc = require('../src/nem/models');
+const sc = require('../src/symbol/models');
+const { hexToUint8 } = require('../src/utils/converter');
+const YAML = require('yaml');
+const yargs = require('yargs');
+const fs = require('fs');
+const path = require('path');
+
+(() => {
+	const areEqual = (lhs, rhs) => {
+		if (!!lhs !== !!rhs)
+			return false;
+
+		if (lhs.length && rhs.length) {
+			if (lhs.length !== rhs.length)
+				return false;
+
+			for (let i = 0; i < lhs.length; ++i) {
+				if (!areEqual(lhs[i], rhs[i]))
+					return false;
+			}
+
+			return true;
+		}
+
+		if (lhs.bytes && rhs.bytes)
+			return areEqual(lhs.bytes, rhs.bytes);
+
+		return lhs === rhs;
+	};
+
+	const assertRoundtripBuilder = (module, testCase) => {
+		// Arrange:
+		const payload = hexToUint8(testCase.payload);
+		const builder = module[testCase.schema_name];
+		if (!builder)
+			throw RangeError(`invalid builder name: ${testCase.schema_name}`);
+
+		// Act:
+		const transaction = builder.deserialize(payload);
+
+		console.log(transaction.toString());
```

Comment:
intended?

Reply:
_partially_, intent was to trigger toString, doesn't have to go to console obviously

Reply:
would at least add comment for reason